bazel/benchmark/driver-utilities/perf_util.ts

```diff
@@ -71,16 +75,13 @@ function createBenchpressRunner(): Runner {
   if (process.env.GIT_SHA) {
     runId = process.env.GIT_SHA + ' ' + runId;
   }
-  const resultsFolder = './dist/benchmark_results';
```

Comment:
Is there a reason that we no longer need to create this directory?  I know that bazel should already provide the `TEST_UNDECLARED_OUTPUTS_DIR` value, but if it doesn't we need to make sure that the directory we use exists still correct?

Reply:
good point. is it even worth caring about the non-bazel case when this utility is part of `/bazel`

Reply:
Probably not, I think its fine just operate on the assumption that the environment value will be there.  Or to error if its not present instead of falling back.